test/user.test.js

```diff
@@ -205,6 +205,79 @@ describe('User', function() {
       });
     });
 
+    describe('Password length validation', function() {
+      var pass72 = 'ThePasswordShouldNotExceedtheMaximumNumberOfcharactersAndItisInvalidIfIt';
+      var pass73 = 'ThePasswordShouldNotExceedtheMaximumNumberOfcharactersAndItisInvalidIfIt1';
+      var badPass = 'ThePasswordShouldNotExceedtheMaximumNumberOfcharactersAndItisInvalidIfItXYZ1';
+
+      it('rejects passwords longer than 72 characters', function(done) {
+        try {
+          User.create({ email: 'b@c.com', password: pass73 }, function(err) {
+            if (err) return done (err);
+            expect(err).to.have.property('message', /Password too long/);
+          });
+        } catch (e) {
+          done();
+        }
+      });
+
+      it('rejects a new user with password longer than 72 characters', function(done) {
+        try {
+          var u = new User({ username: 'foo', password: pass73 });
+          assert(false, 'Error should have been thrown');
+        } catch (e) {
+          done();
+        }
+      });
+
+      it('accepts passwords that are exactly 72 characters long', function(done) {
+        User.create({ email: 'b@c.com', password: pass72 }, function(err, user) {
+          if (err) return done(err);
+          User.findById(user.id, function(err, userFound)  {
+            if (err) return done (err);
+            assert(userFound);
+            done();
+          });
+        });
+      });
+
+      it('allows login with password exactly 72 characters long', function(done) {
```

Comment:
This test is nested under `describe('User.create')` right now. Please move it to `describe('User.login')` or perhaps move `describe('Password length validation')` to higher in the hierarchy, right under the top-level `describe('User')`.